app/classifier/tasks/dropdown/index.spec.js

```diff
@@ -1,66 +1,263 @@
+/* eslint-disable func-names, prefer-arrow-callback */
+/* global describe, it, beforeEach */
+
 import React from 'react';
 import assert from 'assert';
 import { mount } from 'enzyme';
+import Select from 'react-select'; // required to properly simulate change
 import DropdownTask from './';
+import { workflow } from '../../../pages/dev-classifier/mock-data';
 
-const task = {
+const singleSelect = {
   instruction: 'Is there something here?',
   selects: [{
-    id: 'numbers',
+    id: 'numbers123',
     title: 'Numbers',
     options: {
       '*': [
-        {label: 'One', value: 1},
-        {label: 'Two', value: 2},
-        {label: 'Three', value: 3}
+        { label: 'One', value: 1 },
+        { label: 'Two', value: 2 },
+        { label: 'Three', value: 3 }
       ]
-    },
-    required: true
+    }
   }]
 };
 
-const annotation = {
-  value: [{option: true, value: 3}]
-};
+const multiSelects = workflow.tasks.dropdown;
 
-describe('DropdownTask', function () {
-  let wrapper;
+// multiSelects:
+//   1 - Country (required:true)
+//   2 - State (condition:Country, required:true, allowCreate:false)
+//   3 - County (condition:State, allowCreate:true)
+//   4 - City (condition:County, allowCreate:false)
+//   5 - Best State Team (condition:State, allowCreate:true)
+
+describe('DropdownTask:static methods', function () {
+  it('should have the correct question text', function () {
+    assert.equal(DropdownTask.getTaskText(singleSelect), singleSelect.instruction);
+  });
 
-  beforeEach(function () {
-    wrapper = mount(<DropdownTask task={task} annotation={annotation} onChange={function(a) {return a;} } />);
+  it('the default annotation should be an empty array', function () {
```

Comment:
Small note on style: "it the default annotation should be…" doesn't read very well so maybe reword this test?